docs/REFERENCE.md

```diff
@@ -93,11 +135,11 @@ async function promiseResolver() {
 
 
 ## Err
-`Err` represents an invalid result of an action.
+An `Err` object represents an invalid result of an action.
 
 ### Err.of(...)
 
-Create new `Err` result. Static function (can be called only on imported `Err` namespace, not on `Err` results).
+Creates a new `Err` result. It is a static function, meaning it can only be called on the imported `Err` namespace and not on `Err` result instances.
```

Comment:
Same information about `Err.of(promise.resolve())`

Reply:
there is no such supported syntax that lead to anything meaningful

Reply:
same argument as for `Ok.of`